lib/galaxy/tools/parser/xml.py

```diff
@@ -483,6 +483,12 @@ def parse_profile(self):
         # - Enable buggy interpreter attribute.
         return self.root.get("profile", "16.01")
 
+    def parse_python_template_version(self):
+        python_template_version = self.root.get("python_template_version", None)
+        if python_template_version is not None:
+            python_template_version = float(python_template_version)
```

Comment:
```suggestion
            python_template_version = packaging.version.parse(python_template_version)
```

You'd need to also add `import packaging.version` at the top.